pkg/virt-controller/services/renderresources.go

```diff
@@ -122,6 +122,10 @@ func WithoutDedicatedCPU(cpu *v1.CPU, cpuAllocationRatio int) ResourceRendererOp
 				vcpusStr = fmt.Sprintf("%gm", val)
 			}
 			renderer.calculatedRequests[k8sv1.ResourceCPU] = resource.MustParse(vcpusStr)
+
+			if withCPULimits {
```

Comment:
What about a case where limits are provided?

Reply:
I think a few tests in `template_test.go` would help

Reply:
Added, PTAL!